pincer/objects/guild/channel.py

```diff
@@ -70,10 +73,11 @@ class ChannelType(IntEnum):
 
     if GatewayConfig.version >= 9:
         GUILD_NEWS_THREAD = 10
-        GUILD_PUBLIC_THREAD = 11
-        GUILD_PRIVATE_THREAD = 12
+        GUILD_THREAD = 11
+        GUILD_PUBLIC_THREAD = 12
+        GUILD_PRIVATE_THREAD = 13
 
-    GUILD_STAGE_VOICE = 13
+    GUILD_STAGE_VOICE = 14
 
```

Comment:
Why?

Reply:
Thought it would be needed. Can revert that change if desired.

Reply:
wdym? it's based on [this](https://discord.com/developers/docs/resources/channel#channel-object-channel-types)

Reply:
Oh whoops... Will revert it then 👍